portal-frontend/src/app/features/public/application/alc-review/alc-review.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { BehaviorSubject, Subject, takeUntil } from 'rxjs';
+import { ApplicationDocumentDto } from '../../../../services/application-document/application-document.dto';
+import { ApplicationSubmissionReviewService } from '../../../../services/application-submission-review/application-submission-review.service';
+import {
+  SUBMISSION_STATUS,
+  ApplicationSubmissionDetailedDto,
+} from '../../../../services/application-submission/application-submission.dto';
+
+@Component({
+  selector: 'app-alc-review',
+  templateUrl: './alc-review.component.html',
+  styleUrls: ['./alc-review.component.scss'],
+})
+export class AlcReviewComponent implements OnInit, OnDestroy {
+  private $destroy = new Subject<void>();
+
+  @Input() $application = new BehaviorSubject<ApplicationSubmissionDetailedDto | undefined>(undefined);
+  @Input() $applicationDocuments = new BehaviorSubject<ApplicationDocumentDto[]>([]);
+
+  application: ApplicationSubmissionDetailedDto | undefined;
+  SUBMISSION_STATUS = SUBMISSION_STATUS;
+
+  constructor(private applicationReviewService: ApplicationSubmissionReviewService, private router: Router) {}
+
+  ngOnInit(): void {
+    this.$application.pipe(takeUntil(this.$destroy)).subscribe((application) => {
+      this.application = application;
+    });
+  }
+
+  ngOnDestroy(): void {
+    this.$destroy.next();
+    this.$destroy.complete();
+  }
+
+  async onReview(fileId: string) {
+    if (this.application?.status.code === SUBMISSION_STATUS.SUBMITTED_TO_LG) {
+      const review = await this.applicationReviewService.startReview(fileId);
+      if (!review) {
+        return;
+      }
+    }
+    await this.router.navigateByUrl(`application/${fileId}/review`);
```

Comment:
is this a correct url? Are we agreed on '/public' or simply '/' for public urls?

Reply:
Sorry! This is the plain copy paste version, same as the L/FNG components. I haven't updated them yet and they are currently commented out.